packages/app/src/cli/models/extensions/specifications/flow_action.ts

```diff
@@ -28,6 +33,25 @@ const FlowActionExtensionSchema = BaseSchema.extend({
   }),
 })
 
+/**
+ * Loads the schema from the partner defined file.
+ */
+const loadSchemaPatchFromPath = async (extensionPath: string, patchPath: string | undefined) => {
+  if (!patchPath) {
+    return ''
+  }
+
+  const path = await glob(joinPath(extensionPath, patchPath))
+
+  if (path.length > 1) {
+    throw new Error('Multiple files found for schema patch path')
+  } else if (path.length === 0) {
+    return ''
+  }
+
+  return fs.readFileSync(path[0] as string, 'utf8')
```

Comment:
I used `readFileSync` because I've seen it in the [locale loader](https://github.com/Shopify/cli/blob/23c5dcacb00907302a7de47010daa5b79fa52db1/packages/app/src/cli/utilities/extensions/locales-configuration.ts) for checkout UI extensions. But I do also see `readFile` used at times 🤔

Reply:
Any of those is OK, but import it from `'@shopify/cli-kit/node/fs'` and not from `fs` directly please! :)